packages/myst-frontmatter/src/project/types.ts

```diff
@@ -5,6 +5,14 @@ import type { Numbering } from '../numbering/types.js';
 import type { SiteFrontmatter } from '../site/types.js';
 import type { Thebe } from '../thebe/types.js';
 
+type OutputRemovalOptions = 'show' | 'remove' | 'remove-warn' | 'remove-error' | 'warn' | 'error';
+
+export type ProjectSettings = {
+  output_stderr?: OutputRemovalOptions;
```

Comment:
The attribute in jupyterbook is reversed `stderr_output` - any value to matching that? https://jupyterbook.org/en/stable/content/execute.html#dealing-with-code-that-produces-stderr (The `settings` key and everything is different, so not sure how much it matters)

Reply:
Added aliases!
It is the opposite in mystnb...!